core-services/prow/02_config/rh-ecosystem-edge/NVIDIA-gpu-add-on-init-container/_prowconfig.yaml

```diff
@@ -0,0 +1,22 @@
+branch-protection:
+  orgs:
+    rh-ecosystem-edge:
+      repos:
+        NVIDIA-gpu-add-on-init-container:
+          protect: false
+tide:
+  merge_method:
+    rh-ecosystem-edge/NVIDIA-gpu-add-on-init-container: rebase
```

Comment:
seems like the most popular option is ``squash``, any reason to use a different method in this repo?

Reply:
Not really. I prefer it, But I get why this might be popular :)

Reply:
> Not really. I prefer it

+1 or even +100 :) I spend <del>hours</del> a significant time shaping my commits into logical layers, and get angry when I see all of them squashed into a single "merge branch abc" -,-

Reply:
I can say I have a different preference
But surely you're the ones working with this repo, so feel free to configure it as you wish :smiley: